js/main.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+{
+  const add_btn = document.getElementById('add_btn');
```

Comment:
これまでに指摘ができていなかったら大変申し訳ないのですが…
javascript は `camelCase` にする慣習があり `snake_case` はあまり用いられません。
https://wa3.i-3-i.info/diff71moji.html

```suggestion
  const addBtn = document.getElementById('add_btn');
```